test/regress/CMakeLists.txt

```diff
@@ -1108,6 +1108,8 @@ set(regress_0_tests
   regress0/strings/issue5428-re-diff-assoc.smt2
   regress0/strings/issue5542-strings-seq-mix.smt2
   regress0/strings/issue5608-eager-pp.smt2
+  regress0/strings/issue5666-orig-unit-deq.smt2
+  regress0/strings/issue5666-unit-deq.smt2
```

Comment:
do you think there is any way to trigger this bug for pure strings (e.g. with strings of length 1)? If no, never mind. If yes, maybe you can add a regression?

Reply:
No, this is specific to the case of seq.unit.  (Disequalities between constant strings of length one are trivially satisfied).